### PR TITLE
Add s3-file-field

### PR DESCRIPTION
**Pull request by dchiquit**
The default `Image` model now uses an `S3FileField` to store the file.

The `README.md` is still missing one step, creating the `django-storage`
bucket in the Minio instance. This needs to be either added to the set
up instructions (which would be a rather obnoxious step during initial
setup) or the `girder/minio-nonroot` image should include that bucket by
default.

### CODE REVIEW

File: {{ cookiecutter.project_slug }}/README.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ but allows developers to run the Python code on their native system.
    2. `celery worker --app {{ cookiecutter.pkg_name }}.celery --loglevel info --without-heartbeat`
 2. When finished, run `docker-compose stop`
 
+## TODO Delete the example migration!
```
Comment (brianhelba) [inline {{ cookiecutter.project_slug }}/README.md:43]: Can you move this text up a level to the top-level README? It's more of meta-info about how to use the cookiecutter, rather than how to use the application that the cookiecutter generates.

File: {{ cookiecutter.project_slug }}/{{ cookiecutter.pkg_name }}/settings.py [hunk 2]
```diff
@@ -20,17 +19,13 @@ class {{ cookiecutter.pkg_name.split('_')|map('capitalize')|join('') }}Config(Co
...
     def before_binding(configuration: ComposedConfiguration) -> None:
-        configuration.INSTALLED_APPS += ['{{ cookiecutter.pkg_name }}.{{ cookiecutter.first_app_name }}.apps.{{ cookiecutter.first_app_name.split('_')|map('capitalize')|join('') }}Config']
+        configuration.INSTALLED_APPS += ['{{ cookiecutter.pkg_name }}.{{ cookiecutter.first_app_name }}.apps.{{ cookiecutter.first_app_name.split('_')|map('capitalize')|join('') }}Config', 's3_file_field']
 
 
 class DevelopmentConfiguration({{ cookiecutter.pkg_name.split('_')|map('capitalize')|join('') }}Config, DevelopmentBaseConfiguration):
     pass
 
 
-class TestingConfiguration({{ cookiecutter.pkg_name.split('_')|map('capitalize')|join('') }}Config, TestingBaseConfiguration):
```
Comment (brianhelba) [inline {{ cookiecutter.project_slug }}/{{ cookiecutter.pkg_name }}/settings.py:30]: Why is this deleted?

**Comment by brianhelba** [comment]
Do you know why the bucket isn't created? It was getting auto-created previously, without S3FF.

**Comment by brianhelba** [comment]
Additionally, can you verify that both upload and download work when this is run in full-Docker mode? I believe uploading to a server in Docker doesn't work. If you can reproduce the problem, I have an idea about the fix, which I'm happy to discuss.

File: {{ cookiecutter.project_slug }}/{{ cookiecutter.pkg_name }}/settings.py (hunk 2 above)
Comment (dchiquit) [reply {{ cookiecutter.project_slug }}/{{ cookiecutter.pkg_name }}/settings.py]: I thought it had been deleted from `django-girders`, turned out my copy was just out of date. It is restored.

**Comment by dchiquit** [comment]
Upload does not work using docker-compose. I can fix that if you know the cause.

I have no idea why my bucket isn't being created. All the config options are set correctly, the only way I can imagine it failing is if [the check](https://github.com/py-pa/django-minio-storage/blob/0588e44c23035eab32a9619b79157f993503a1cc/minio_storage/storage.py#L87:L92) isn't being called for some reason.
